R/consent.R

```diff
@@ -79,6 +79,7 @@ renv_consent_check <- function() {
     !is.na(Sys.getenv("CI", unset = NA)) ||
     !is.na(Sys.getenv("GITHUB_ACTION", unset = NA)) ||
     !is.na(Sys.getenv("RENV_PATHS_ROOT", unset = NA)) ||
+    dir.exists("/.singularity.d") ||
```

Comment:
Could we use `file.exists()` here instead? (Just because `dir.exists()` was added in R 3.2.0 and, at least in theory, some might be using `renv` with an older verison of R.)

Reply:
Sure, good catch.